### ISSUE TITLE
Could not resolve all artifacts for configuration with AtomicFu "0.25.0"

### ISSUE THREAD

**Issue by amal** [Kotlin/kotlinx-atomicfu#447]
With v0.25.0 and plugin connected by id as `"org.jetbrains.kotlinx.atomicfu"` I receive a project configuration error:

```
> Could not resolve all artifacts for configuration ':fluxo-io-rad:classpath'.
   > Could not find org.jetbrains.kotlinx:atomicfu-transformer:0.25.0.
     Searched in the following locations:
       - https://plugins.gradle.org/m2/org/jetbrains/kotlinx/atomicfu-transformer/0.25.0/atomicfu-transformer-0.25.0.pom
     Required by:
         project :fluxo-io-rad > org.jetbrains.kotlinx.atomicfu:org.jetbrains.kotlinx.atomicfu.gradle.plugin:0.25.0 > org.jetbrains.kotlinx:atomicfu-gradle-plugin:0.25.0
```

Seems like `atomicfu-transformer` artifact wasn't published both in gradle plugins portal and maven central.

**Comment by mvicsokolova** [comment]
Thank you!
Publication failed for some reason, I've republished all the artifacts now. They should appear at `mavenCentral` soon.

**Comment by eygraber** [comment]
I'm not getting warnings about the artifact not being present anymore, but I get unresolved reference errors for `kotlinx.atomicfu*` with just the plugin applied. I can only get them to resolve by manually including the runtime dependency (`implementation("org.jetbrains.kotlinx:atomicfu:0.25.0")`). Is that expected?

**Comment by mvicsokolova** [comment]
No, this is not expected. Could you please share the part of the build script, where you apply the plugin? 👀

Do you apply it by the new id? Like this

```
plugins {
     id("org.jetbrains.kotlinx.atomicfu") version "0.25.0"
}
```

**Comment by amal** [comment]
Checked, [seems](https://github.com/fluxo-kt/fluxo-io/pull/4/files) to work now.

**Comment by eygraber** [comment]
> No, this is not expected. Could you please share the part of the build script, where you apply the plugin?

I made a copy paste error. It's working fine, sorry about that.
